### DISCUSSION TITLE
Guida uMap

### DISCUSSION THREAD

**Discussion by aborruso** [opendatasicilia/comunita#5] [Idee]
È un software che usiamo molto, di cui spesso mi rendo conto che manchi una guida.

Inizio a creare un repo e poi se @cirospat @gbvitrano @pigreco vogliono dare una mano ...

Lo farei basato sul mitico [Material for MkDocs](https://squidfunk.github.io/mkdocs-material/)

**Comment by aborruso** [comment]
Il repo https://github.com/opendatasicilia/guida-umap

**Reply by cirospat** [reply]
la guida ha ora i contenuti ed un bel vestito

**Reply by aborruso** [reply]
Applausi al grande @cirospat !!

**Reply by aborruso** [reply]
L'URL è https://opendatasicilia.github.io/guida-umap/
